src/github.com/t3chguy/riot-static/mxclient/public-rooms.go

```diff
@@ -1,84 +0,0 @@
-// Copyright 2017 Michael Telatynski <7t3chguy@gmail.com>
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package mxclient
-
-import (
-	"github.com/matrix-org/gomatrix"
-	"github.com/t3chguy/riot-static/utils"
-	"sync"
-)
-
-type WorldReadableRooms struct {
-	mxclient *Client
-	sync.RWMutex
```

Comment:
Gee, I sure wish I knew what this was protecting. I would've been happy with one of:
 - A comment.
 - An obvious variable name like `roomsMutex` rather than it being embedded like this.